src/petals/client/remote_forward_backward.py

```diff
@@ -108,7 +108,7 @@ async def run_remote_forward(
 
     # call RPC on remote server
     size = sum(t.element_size() * t.nelement() for t in inputs)
-    forward_fn = _forward_stream if size > MAX_UNARY_PAYLOAD_SIZE else _forward_unary
+    forward_fn = _forward_stream if size > MAX_UNARY_PAYLOAD_SIZE // 2 else _forward_unary
```

Comment:
[nit] please check how MAX_UNARY_PAYLOAD_SIZE is defined - perhaps it's best to use MAX_MESSAGE_SIZE//4?